### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by Umuhan**
### Below is the requirement for our project.

- [X] A Navigation bar.

- [X] Three content sections:

1. Books list.

2. Add book form.

3. Contact info.

### CODE REVIEW

File: CSS/styles.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  margin: 0 auto;
+  text-align: center;
+  font-size: 2.5rem;
+}
+
+/* .all-books {
+  margin-top: 2rem;
+  width: 50vw;
+  border: 1px solid #000;
+} */
```
Comment (abdulhamiid) [inline CSS/styles.css]: Please kindly check all your files, `adjust.js`, and `index.js`, and remove all commented codes 👍 

Please, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Understand the concept of a single-page application.
- ✔️ Use JavaScript to manipulate DOM elements.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/styles.css (hunk 1 above)
Comment (Whoistolu) [reply CSS/styles.css]: - Well resolved team 👍🏽

File: app.js/adjust.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-unused-vars */
```
Comment (Whoistolu) [inline app.js/adjust.js:1]: - [ ] [OPTIONAL] Let's avoid disabling eslint in order for it to help standardize our JS code. Kindly resolve this issue manually.

File: app.js/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-unused-vars */
```
Comment (Whoistolu) [inline app.js/index.js:1]: - [ ] [OPTIONAL] Let's avoid disabling eslint in order for it to help standardize our JS code. Kindly resolve this issue manually.

**Review by Whoistolu — APPROVED** [review]
### Hi Team,

Great job implementing the recommended fixes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**